dissect/target/plugins/os/windows/tasks.py

```diff
@@ -58,452 +61,114 @@
-                email_bcc = self.get_element("Actions/ShowMessage/Bcc")
-                email_reply_to = self.get_element("Actions/ShowMessage/ReplyTo")
-                email_from = self.get_element("Actions/ShowMessage/From")
-                email_body = self.get_element("Actions/ShowMessage/Body")
-                email_attachments = self.get_element("Actions/ShowMessage/Attachments/File")
-                email_headers_name = self.get_element("Actions/ShowMessage/Headers/HeadersField/Name")
-                email_headers_value = self.get_element("Actions/ShowMessage/Headers/headersField/Value")
-                yield SendEmailRecord(
-                    action_type=action_type,
-                    server=email_server,
-                    subject=email_subject,
-                    to=email_to,
-                    cc=email_cc,
-                    bcc=email_bcc,
-                    replyto=email_reply_to,
-                    email_from=email_from,
-                    header_name=email_headers_name,
-                    header_value=email_headers_value,
-                    body=email_body,
-                    attachment=email_attachments,
-                )
-
-            if action_type == "ShowMessage":
-                title = self.get_element("Actions/ShowMessage/Title")
-                body = self.get_element("Actions/ShowMessage/Body")
-                yield ShowMessageRecord(
-                    action_type=action,
-                    title=title,
-                    body=body,
-                )
-
 
 class TasksPlugin(Plugin):
-    """TODO."""
+    """Plugin for retrieving scheduled tasks on a Windows system.
+
+    Args:
+        target: The target system.
+    """
+
+    ATTRIBUTES = [
+        "uri",
+        "security_descriptor",
+        "source",
+        "date",
+        "last_run_date",
+        "author",
+        "version",
+        "description",
+        "documentation",
+        "principal_id",
+        "user_id",
+        "logon_type",
+        "display_name",
+        "run_level",
+        "process_token_sid_type",
+        "required_privileges",
+        "allow_start_on_demand",
+        "restart_on_failure_interval",
+        "restart_on_failure_count",
+        "mutiple_instances_policy",
+        "dissalow_start_on_batteries",
+        "stop_going_on_batteries",
+        "start_when_available",
+        "network_profile_name",
+        "run_only_network_available",
+        "wake_to_run",
+        "enabled",
+        "hidden",
+        "delete_expired_task_after",
+        "idle_duration",
+        "idle_wait_timeout",
+        "idle_stop_on_idle_end",
+        "idle_restart_on_idle",
+        "network_settings_name",
+        "network_settings_id",
+        "execution_time_limit",
+        "priority",
+        "run_only_idle",
+        "unified_scheduling_engine",
+        "disallow_start_on_remote_app_session",
+        "data",
+    ]
```

Comment:
Similar attributes can be retrieved from `TaskRecord.fields.keys()`, then you wouldn't need to duplicate them.